public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+- This does NOT mean that new validators cannot join, but removes the exponential climb in how many are allowed to join at a time
+
+## Will all transactions on L2s now use temporary blob space, or will you be able to choose? {#calldata-vs-blobs}
+
+Rollup transactions are not required to use blob space for their data (though they are economically incentivized to do so). Each can optionally write their data to blobs (guaranteed available for all challenge periods, very cheap) or to smart contract "calldata" (permanent storage, more expensive). Rollup providers will decide which to use based on the current demand for blob space. If blobs are in high demand, the rollup may decide to use calldata to ensure the data is posted in a timely manner.
+
+Although this decision is likely to be performed behind-the-scenes by rollup providers, it is theoretically possible for a user to choose which storage type to use. This would require the rollup provider to expose this option to the user, which introduces additional complexities when bundling transactions in a cost-effective manner. See individual rollup provider documentation for more details.
+
+## Would this reduce layer 1 (L1) gas fees in any way? {#l1-fee-impact}
+
+Yes, but significantly less than on layer 2 rollups. A new gas market is introduced exclusively for blob space, for use by rollup providers. This reduces the demand from these rollups for L1 block space, pushing gas prices downward.
+
+_Although fees on L1 may be reduced by off-loading rollup data to blobs, this upgrade primarily focuses on the reduction of L2 fees. Reduction of fees on L1 (Mainnet) may occur as a second-order effect to a lesser extent._
+
+- L1 gas reduction will be proportional to adoption/usage of blob data by rollup providers
+- L1 gas is likely to remain competitive from non-rollup related activity
+- Rollups that adopt the use of blob space will demand less L1 gas, helping push L1 gas fees downward in the near-term
+- Blob space is still limited, so if blobs within a block are saturated/full, then rollups may be required to post their data as permanent data in the meantime, which would drive L1 and L2 gas prices up
+
+## Will this reduce fees on other EVM layer 1 blockchains? {#alt-l1-fee-impact}
+
+No. The benefits of Proto-Danksharding are specific to Ethereum layer 2 rollups that store their proofs on layer 1 (Mainnet).
+
+Simply being compatible with the Ethereum Virtual Machine (EVM) does not mean that a network will see any benefit from this upgrade. Networks that operate independently of Ethereum (whether EVM compatible or not), such as Polygon PoS, Binance Smart Chain, Solana, Avalanche, or Gnosis Chain, do not store their data on Ethereum and will not see any benefit from this upgrade.
+
+[More about layer 2 rollups](/layer-2/)
+
+## More of a visual learner? {#visual-learner}
+
+<YouTube id="HT9PHWloIiU" />
```

Comment:
its a bad practice to put video at the bottom of the page, should be higher, basically almost at the top. 

Also the headline "More of a visual learner?" is redundant

Reply:
Those who would prefer video content likely might not even reach the end of the page. As a user I would like to have the choice how to consume content at top (consume by reading vs consume by watching a video)

Reply:
The header and position is in alignment with the rest of the repo

Reply:
Happy to change, just noting this is a pattern previously established

Reply:
I don't see this as blocking. Agree w/@konopkja that we should reconsider the pattern, but agree w/@wackerow that we're just using existing patterns and this PR might not be the best place to reconsider those.

Reply:
we have already reconsidered that and changed it on few pages during comprehension small edits btw, but agree not a blocker here